docs/acme-integration.rst

```diff
@@ -156,28 +156,44 @@ configuration active, it will check for validity of the certificate, and
 about a month before the expiration date it will try to renew the certificate
 automatically.
 
-Certificate for subdomains excluding the apex domain
-----------------------------------------------------
+Example: Certificate for apex domain and subdomains
```

Comment:
If you take `example.com` DNS domain with `www.example.com`, `smtp.example.com` subdomains, an "apex domain" would be the `example.com` itself. Typically this is redirected to the homepage of the domain. I suppose that the "apex domain" could be less known than alternative "root domain", but the naming space in this context is very limited so I prefer to use a different name for this - "root domain" could be understood as the `.` DNS domain.

Some links that use the term:

- https://stackoverflow.com/questions/16022324/how-to-setup-dns-for-an-apex-domain-no-www-pointing-to-a-heroku-app
- https://blog.cloudflare.com/zone-apex-naked-domain-root-domain-cname-supp/